python/TestHarness/testers/RunApp.py

```diff
@@ -282,15 +282,10 @@ def getCommand(self, options):
         if options.scaling and specs['scale_refine'] > 0:
             cli_args.insert(0, ' -r ' + str(specs['scale_refine']))
 
-        if specs.isValid('capabilities'):
-            if 'libtorch' in specs['capabilities']:
-                cli_args.append(f'--libtorch-device {options.device}')
-            if 'mfem' in specs['capabilities']:
-                cli_args.append(f'Executioner/device={options.device}')
-
         # Get the number of processors and threads the Tester requires
         ncpus = self.getProcs(options)
         nthreads = self.getThreads(options)
+        cli_args.append(f'--compute-device {options.device}')
```

Comment:
```suggestion
        cli_args.append(f'--compute-device={options.device}')
```

Not required, personal preference after dealing with splitting chaos